webui/react/src/pages/TrialDetails/Header/TrialHeaderLeft.tsx

```diff
@@ -26,6 +32,21 @@ const TrialHeaderLeft: React.FC<Props> = ({ experiment, trial }: Props) => {
       <div className={css.trial}>
         <ExperimentIcons state={trial.state} />
         <div>Trial {trial.id}</div>
+        {trial.logSignal &&
+          (trial.logSignal.length < labelMaxLength ? (
+            <Badge
+              backgroundColor={hex2hsl(labelColor)}
+              key={trial.logSignal}
```

Comment:
is `logSignal` always unique?

Reply:
You are right, it's not always unique, but `key` is unnecessary and I should just remove it